File: MayorMenor.f90
program mayor_menor
    implicit none
    integer :: numero, referencia

    write(*,*) 'Ingrese el número de referencia:'
    read(*,*) referencia

    write(*,*) 'Ingrese un número para comparar:'
    read(*,*) numero

    if (numero > referencia) then
        write(*,*) 'El número ', numero, ' es MAYOR que ', referencia
    else if (numero < referencia) then
        write(*,*) 'El número ', numero, ' es MENOR que ', referencia
    else
        write(*,*) 'El número ', numero, ' es IGUAL a ', referencia
    end if


    write(*,*) 'Presione ENTER para continuar...'
    read(*,*)

end program mayor_menor

File: MayorMenor.cpp
#include <stdio.h>

int main() {
    int numero, referencia;
    
   
    printf("Ingrese el número de referencia: ");
    scanf("%d", &referencia);
    
   
    printf("Ingrese un número para comparar: ");
    scanf("%d", &numero);
    
    
    if (numero > referencia) {
        printf("\nEl número %d es MAYOR que %d\n", numero, referencia);
    } else if (numero < referencia) {
        printf("\nEl número %d es MENOR que %d\n", numero, referencia);
    } else {
        printf("\nEl número %d es IGUAL a %d\n", numero, referencia);
    }
    
    return 0;
}
